tensorboard/data/server/gcs/logdir.rs

```diff
@@ -79,24 +80,45 @@ pub struct Logdir {
     /// Invariant: `prefix` either is empty or ends with `/`, and thus an event file name should be
     /// joined onto `prefix` to form its full object name.
     prefix: String,
+    /// Size of the opened file read buffer (in Kb) when reading from GCS.
+    /// The `gcs::Logdir::new` will attempt to fetch the `TB_GCS_BUFFER_SIZE_KB` environment
+    /// variable that represent the read buffer size (in Kb) for each TF events file.
+    /// Note: if reading a large number of TF events files, set an appropriate value for
+    /// `buffer_capacity` to prevent running out of memory. This determines the total size of the
+    /// allocated memory.
+    /// The default value is defined by the `DEFAULT_BUFFER_CAPACITY` constant.
```

Comment:
Nit: Update the reference to `DEFAULT_BUFFER_CAPACITY_KB` to match the variable name.

Reply:
Sorry for that...

Reply:
No worries, we very much appreciate your contribution!